server/cache.go

```diff
@@ -400,7 +400,7 @@ func (c *clusterInfo) handleRegionHeartbeat(region *core.RegionInfo) error {
 
 	if saveKV && c.kv != nil {
 		if err := c.kv.SaveRegion(region.Region); err != nil {
-			return errors.Trace(err)
+			log.Errorf("[region %d] fail to save kv: %v", region.GetId(), err)
```

Comment:
fail to save region %v: err: %v

Reply:
should add a comment that why we can still go on even the save failed.

Reply:
All region related logs are using the format '[region xx] xxx', which is grep friendly.

Reply:
sorry, I meant `[region %d] fail to save region %v: err: %v`.